google_speech/__init__.py

```diff
@@ -119,8 +119,10 @@ def cleanSpaces(dirty_string):
     return __class__.CLEAN_MULTIPLE_SPACES_REGEX.sub(" ",
                                                      dirty_string.replace("\n", " ").replace("\t", " ").strip())
 
-  def play(self, sox_effects=()):
-    """ Play a speech. """
+  def __buildSegments__(self):
```

Comment:
`__xxx__` method names are reserved for Python's "magic methods". `buildSegments` is fine.